<launch>
	<group ns="robot_0">
		<node pkg="se306project" name="pubnode" type="PickerRobot"/>
	</group>

	<group ns="robot_1">
		<node pkg="se306project" name="pubnode" type="PickerRobot"/>
	</group>
</launch>
